src/libraries/System.Private.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/JSObject.cs

```diff
@@ -164,7 +164,7 @@ internal void FreeHandle()
             FreeGCHandle();
         }
 
-        public override bool Equals(object? obj) => obj is JSObject other && JSHandle == other.JSHandle;
+        public override bool Equals([NotNullWhen(true)] object? obj) => obj is JSObject other && JSHandle == other.JSHandle;
```

Comment:
Seems forgot to import System.Diagnostics.CodeAnalysis

Reply:
Thanks